templates/server.conf.j2

```diff
@@ -54,7 +54,7 @@ ifconfig-pool-persist {{ openvpn_ifconfig_pool_persist_file }}
 client-config-dir {{ openvpn_client_config_dir }}
 {% endif %}
 
-{% if openvpn_redirect_gateway is defined %}
+{% if openvpn_redirect_gateway is defined and openvpn_redirect_gateway != '' %}
```

Comment:
I think this could be simplified to `{% if openvpn_redirect_gateway is truthy %}`, but this is more restrictive in what it accepts